src/types/FormattedLoginOption.js

```diff
@@ -8,6 +8,7 @@ const FormattedLoginOption = PropTypes.shape({
     title: PropTypes.string.isRequired,
     imageSrc: PropTypes.string.isRequired,
     href: PropTypes.string,
+    appearance: PropTypes.bool,
```

Comment:
```suggestion
    appearance: PropTypes.oneOf(LOGO_APPEARANCES).isRequired,
```
you can <kbd>CTRL</kbd> + <kbd>F</kbd> on this line and you'll find where this is defined. Probably it would be useful to create `src/types/LoginOptionLogo.js` to put this struct, and then do a search on `imageSrc` in the codebase since this shape has been duplicated in a number of places - let's re-use it as a custom proptype instead.

Reply:
No idea what happened in my head when I wrote `bool` :thinking: will change this!